scripts/compare_pyspark_and_pysparkling_public_interface.py

```diff
@@ -59,67 +65,211 @@ def tell_files_to_be_internalized():
         log.info('- %s', file)
 
 
+def compare_one_symbol(txt: str, pyspark_class, pysparkling_class, attr: inspect.Attribute):
+    txt_to_report = f'    - [{txt}] {pyspark_class.__name__}.'
+    if txt in ['ADD', 'CORRECT']:
+        txt_to_report += attr.name
+        if attr.kind == 'data':  # Plain attribute.
+            pass
+        elif attr.kind == 'property':
+            txt_to_report += ' @property'
+        else:
+            txt_to_report += str(inspect.signature(getattr(pyspark_class, attr.name)))
+            if attr.kind == 'static method':
+                txt_to_report += ' @staticmethod'
+            if attr.kind == 'class method':
+                txt_to_report += ' @classmethod'
+    elif txt == 'DEL':
+        txt_to_report += attr.name
+    else:
+        raise NotImplementedError
+
+    log.info(txt_to_report)
+
+
+def compare_symbols(pyspark_symbol, pysparkling_symbol):
```

Comment:
`compare_symbols` looks good, but it's tricky to verify its correctness (same comment for remaining functions from this module). 
Manual runs will help us with additional fine-tuning :)

Reply:
Yeah, it's not meant to be readable. It's just meant to give a quick overview if it's correct what we're doing :-). If anything is found that doesn't look correct, need to adjust this.

Anyway, all methods should be coming under test!